deployments/fpga_admissionwebhook/af-crd.yaml

```diff
@@ -20,3 +20,9 @@ spec:
             afuId:
               type: string
               pattern: '^[0-9a-f]{8,128}$'
```

Comment:
nit: this might be overkill. we don't expect such long UUIDs.

Reply:
Limited to 32.